SwiftDashboard.md

```diff
@@ -0,0 +1,56 @@
+# Firebase Swift Modernization Dashboard
+
+This dashboard summarizes the status of Firebase's [2022 Swift Modernization Project](Roadmap.md).
+Please upvote or create a [feature request](https://github.com/firebase/firebase-ios-sdk/issues)
+to help prioritize any particular cell(s).
+
+|                       | AB  | An     | ApC    | ApD    | Aut    | Cor    | Crs    | DB     | Fst    | Fn     | IAM    | Ins    | Msg    | MLM    | Prf    | RC     |    Str |
```

Comment:
Two other things I can think of that may make sense to track:

1.  SwiftUI improvements/support/enhancements
  - Specifically where there is some element of UI, or dealing with UIViewControllers. This could also encompass some Combine work (publishers can fit in nicely with SwiftUI)

2. Swift Generics
  - This sort of contradicts my other comment... but if we have APIs that are using weakly typed information we should try to use proper generics. Not sure if it's worth having a row for this though.

Reply:
For 1, can you give an example or two.  Is there a more up to date reference than https://docs.google.com/document/d/1UrC4eqwSLG00nSH6V5ttwTkcQt5ULQKkKuiZbtsAGGA/edit?resourcekey=0-sEstblfahgcurSGOzrteXw#heading=h.hdi9c76jzlbq?

For 2, do you have any examples in mind?  If it's just one or two, it might be better in the Other Projects section.

Reply:
1: At a quick glance AppDistro, Auth, FIAM, Perf, and Analytics still have APIs that include UIViewControllers (or implementations that depend on them) which should have an associated API or implementation that work with SwiftUI.  This will overlap with Property Wrappers and likely the SwiftUI lifecycle bits, but an audit and improvements could likely be made. The existing FIAM and Analytics View modifier APIs would fit into this category.

2. Other projects sounds like a good place for this, good call.